### PR TITLE
add Swish op

### PR DESCRIPTION
**Pull request by Douglas-Nyberg**
Adds the Swish activation operator ([spec](https://onnx.ai/onnx/operators/onnx__Swish.html)), introduced in opset 24 ([ONNX 1.19](https://github.com/onnx/onnx/wiki/Logistics-for-ONNX-Release-1.19.0)).

Swish(x, alpha) = x * sigmoid(alpha * x)

ONNX has external test, test_swish_cpu also in opset 24. For now I've added a regression test.

Note: Implemented separately from Tensor.swish() to support the alpha parameter.

### CODE REVIEW

**Comment by geohot** [comment]
Instead of writing this one off regression tests, can we upgrade to onnx 1.19? There's no real reason it's pinned to 1.18.1, and we shouldn't be writing tests we have to maintain.

**Comment by Douglas-Nyberg** [comment]
Happy to upgrade to 1.19. Should I add excludes for the new opset  tests that aren't implemented yet?

**Comment by Douglas-Nyberg** [comment]
To upgrade to ONNX 1.19 I needed to upgrade TensorFlow to 2.19, add exclusions to the newly added attention tests in ONNX 1.19 and remove the CI test for external_test_optim since [TensorFlow Addons](https://www.tensorflow.org/addons/overview) is deprecated.

What would you like done with external_test_optim.py since TF Addons is deprecated?

**Comment by geohot** [comment]
Can you remove tfa in a separate PR and find a different way to test LAMB?

**Comment by Douglas-Nyberg** [comment]
Yea totally, I can remove tfa in external_test_optim.py and try to find a new way to test LAMB. I'll do that in a separate PR. Ill get started on that tomorrow.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -459,7 +459,7 @@ jobs:
       - name: Test Beam Search
         run: CL=1 IGNORE_BEAM_CACHE=1 python3 -m pytest extra/optimization/test_beam_search.py
       - name: Test MLPerf stuff
-        run: CL=1 python -m pytest -n=auto test/external/external_test_optim.py test/external/external_test_losses.py test/external/external_test_metrics.py test/external/external_test_datasets.py --durations=20
```
Comment (geohot) [inline .github/workflows/test.yml:462]: Why is this test removed?

Comment (Douglas-Nyberg) [reply .github/workflows/test.yml:462]: My bad, the test was removed because tensorflow_addons was deprecated. I fixed that in #13555 (replaced with keras.optimizers.Lamb), and now that it's merged I've added the test back.

File: test/models/test_onnx.py [hunk 2]
```diff
@@ -58,18 +58,6 @@ def run(img):
     print(cls, _LABELS[cls])
     assert "car" in _LABELS[cls] or _LABELS[cls] == "convertible"
 
-  def test_pad_list_value(self):
```
Comment (Douglas-Nyberg) [inline test/models/test_onnx.py:61]: Removed test_pad_list_value: this was a regression test I added in https://github.com/tinygrad/tinygrad/pull/13500, but ONNX 1.19's test suite now covers this case in external_test_onnx_backend.py.
